### PR TITLE
(#1276) Forgotten unit-test

### PR DESCRIPTION
**Pull request by iakunin**
This is a PR with forgotten unit-test for https://github.com/yegor256/cactoos/issues/1276

### CODE REVIEW

**Comment by iakunin** [comment]
@paulodamaso could you have a look, please?

File: src/test/java/org/cactoos/scalar/FirstOfTest.java [hunk 1]
```diff
@@ -53,6 +53,24 @@ public void returnsMatchingValue() {
         );
     }
 
+    @Test
+    public void returnsMatchingValueWithExceptionalFallback() {
+        final int value = 2;
+        MatcherAssert.assertThat(
```
Comment (paulodamaso) [inline src/test/java/org/cactoos/scalar/FirstOfTest.java:59]: @iakunin We're using `cactoos-matchers` `Assertion`  instead of `MatcherAssert.assertThat`

Comment (iakunin) [reply src/test/java/org/cactoos/scalar/FirstOfTest.java]: @paulodamaso sure, it's done.

**Comment by paulodamaso** [comment]
@iakunin Thanks!

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@rultor release tag is `0.45`

**Pull request merged**